### ISSUE TITLE
ghcWithHoogle doesn't work correctly anymore

### ISSUE THREAD

**Issue by basvandijk** [NixOS/nixpkgs#26787]
I just noticed that `ghcWithHoogle` is not indexing the specified Haskell packages anymore:

```
$ $(nix-build -E '(import <nixpkgs> {}).haskellPackages.ghcWithHoogle (hsPkgs: with hsPkgs; [ scientific ])')/bin/hoogle search Scientific
No results found
```

The package is also absent from the index page:

```
$ cat result/share/doc/hoogle/index.html | grep -i scientific
```

I'm on a recent `nixos-17.03-small` but on `master` I'm seeing the same result.

@peti do you have an idea what's going on.

**Comment by peti** [comment]
A fix is underway in 50ed8e43f473d666754cb559cb4d2fe214596932.

**Comment by peti** [comment]
Duplicate of https://github.com/NixOS/nixpkgs/issues/26760.

**Comment by basvandijk** [comment]
Thanks for the quick fix! It would be great if the fix is also cherry-picked on `release-17.03`.

**Comment by peti** [comment]
No problem, I'll do that.
